client/persistent_subscriptions_test.go

```diff
@@ -144,28 +125,26 @@ func Test_CreatePersistentStreamSubscription_AfterDeleting(t *testing.T) {
 	streamID := "someStream"
 	pushEventToStream(t, clientInstance, streamID)
 
-	streamConfig := persistent.SubscriptionStreamConfig{
-		StreamOption: persistent.StreamSettings{
-			StreamName: []byte(streamID),
-			Revision:   persistent.Revision_Start,
-		},
-		GroupName: "Group 1",
-		Settings:  persistent.DefaultSubscriptionSettings,
-	}
-
-	err := clientInstance.CreatePersistentSubscription(context.Background(), streamConfig)
+	options := options.PersistentStreamSubscriptionOptionsDefault()
+	err := clientInstance.CreatePersistentSubscription(
+		context.Background(),
+		streamID,
+		"Group 1",
+		&options,
```

Comment:
Can we avoid passing options as pointers?

Reply:
there are many scenarios that involve using the same options over and over again. You wouldn't mind paying the cost of passing that struct by value here?

Reply:
I don't think such a value object would be a performance issue at all. Regardless, it is not that important if you feel strongly about it

Reply:
I don't, I only know Go for the last 2 months, I'm only asking questions :smile:

Reply:
You can write a [benchmark](https://dave.cheney.net/2013/06/30/how-to-write-benchmarks-in-go) and measure the difference. In general, it's orders of magnitude faster than I/O related to network and other operations. Go is also pretty good in re-using memory for the same values (in other words: objects with the same values can be stored under the same address even if they are separate variables).